gulpfile.js

```diff
@@ -182,12 +186,23 @@ const setupCordova = gulp.series(cordovaPlatformAdd, cordovaPrepare, xcode);
 //////////////////
 //////////////////
 
+function validateBuildEnvironment(cb) {
+  log.info(os.platform());
```

Comment:
nit: remove or add more context, i.e. `Building on host ${os.platform()}`

Reply:
Done in e1f99cb